recipes/simplecosine/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "simplecosine" %}
+{% set version = "1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplecosine-{{ version }}.tar.gz
+  sha256: 216ae262f5e5d2daaa7740296f9308c9fc024e7d7551412c7839065748b3bf02
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - python >=3.6
+
+test:
+  imports:
+    - simplecosine
```

Maybe change to `simplecosine.cosine` since the top level `__init__.py` is empty and thus this doesn't really test the package as is?
```suggestion
    - simplecosine.cosine
```